cmd/server/main.go

```diff
@@ -84,16 +84,15 @@ func main() {
 		}
 	}()
 
+	// ASK do we really need to close each repository?
+	// we close db connection above and in all repos we
+	// do the same
```

Comment:
Probably not since we share the `*sql.DB` instance. It was just a habit to have a `close() error` function on each repository.